app/tsconfig.json

```diff
@@ -1,12 +1,17 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
+    "target": "esnext",
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
This is by default turned on when esModuleInterop is true.

Reply:
thanks, will clean this up